### PR TITLE
Module 6 hooks

### PR DESCRIPTION
**Pull request by SergBlack**

### CODE REVIEW

File: src/components/Modal/index.jsx [hunk 1]
```diff
@@ -65,10 +65,10 @@ const Modal = ({
         <StyledWrapper>
           <StyledModal color={color}>
             <StyledModalHeader>
-              {close && (
+              {toggle && (
```
Comment (mmarkelov) [inline src/components/Modal/index.jsx]: Should it be `toggle` here? As i can understand this is a function

File: src/forms/MovieForm.jsx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  release_date: '',
+  poster_path: '',
+  genres: [],
+  overview: '',
+  runtime: '',
+};
+
+const reducer = (state, action) => {
+  switch (action.type) {
+    case 'RESET_FORM':
```
Comment (mmarkelov) [inline src/forms/MovieForm.jsx:34]: Not an error but usually this kind of strings better set as constants

File: src/helpers/getYear.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { DATE_NOT_SET } from '../constants/textMessages';
+
+export default function getYear(date) {
+  const year = new Date(date).getFullYear();
+  return Number.isNaN(year) ? DATE_NOT_SET : year;
```
Comment (mmarkelov) [inline src/helpers/getYear.js]: Maybe like this?
```js
return year || DATE_NOT_SET;
```

File: src/pages/HomePage/index.jsx [hunk 4]
```diff
@@ -11,26 +24,56 @@ import movies from './movies.json';
...
+  height: 450px;
+  width: 100%;
+  margin: 40px 0 20px 0;
+  background-image: url(${BackgroundImage});
+  background-size: cover;
+  background-repeat: no-repeat;
+`;
+
+const HomePage = () => {
+  const [state, setState] = useState([]);
```
Comment (mmarkelov) [inline src/pages/HomePage/index.jsx]: It’s better to rename `state` to `movies` maybe, cause it's just confusing right now

File: src/components/Modal/index.jsx (hunk 1 above)
Comment (SergBlack) [reply src/components/Modal/index.jsx]: I don't think it's neccessary now. Deleted. Thanks

File: src/forms/MovieForm.jsx (hunk 2 above)
Comment (SergBlack) [reply src/forms/MovieForm.jsx:34]: Will rework when redux will be used)

File: src/helpers/getYear.js (hunk 3 above)
Comment (SergBlack) [reply src/helpers/getYear.js]: Fixed. Thank you)

File: src/pages/HomePage/index.jsx (hunk 4 above)
Comment (SergBlack) [reply src/pages/HomePage/index.jsx]: Agree. Fixed)

**Pull request merged**